File: eAnimalShelter/UI/eanimalshelter_desktop/lib/models/requests/volunteer_activity_update_request.dart
import 'volunteer_activity_insert_request.dart';

class VolunteerActivityUpdateRequest
    extends VolunteerActivityInsertRequest {
 

  VolunteerActivityUpdateRequest({
    required super.title,
    required super.description,
    required super.locationId,
    required super.startDateTime,
    required super.endDateTime,
    required super.maxVolunteers,
    
  });

  @override
  Map<String, dynamic> toJson() => {
    ...super.toJson(),
    
  };
}

File: eAnimalShelter/eAnimalShelter.Model/Requests/VolunteerActivityUpdateRequest.cs
using eAnimalShelter.Model.Enums;

namespace eAnimalShelter.Model.Requests
{
    public class VolunteerActivityUpdateRequest
    {
        public string Title { get; set; } = string.Empty;

        public string Description { get; set; } = string.Empty;

        public int LocationId { get; set; }

        public DateTime StartDateTime { get; set; }

        public DateTime EndDateTime { get; set; }

        public int MaxVolunteers { get; set; }

    }
}
